src/test/scripts/functions/jmlc/scalar-input.dml

```diff
@@ -0,0 +1,25 @@
+#-------------------------------------------------------------
```

Comment:
I believe this file has been pulled in from a previous commit by @deroneriksson due to merging as well.